uber/templates/summary/csv_import.html

```diff
@@ -0,0 +1,45 @@
+{% extends "base-admin.html" %}
+{% block title %}CSV Import{% endblock %}
+{% block content %}
+
+<h2> CSV Table Imports </h2>
+This attempts to import a csv file for the table you select. <strong>Please do not refresh this page.</strong>
+
+<form action="import_model" method="post" enctype="multipart/form-data">
+    <select name="selected_model">
+        {% options tables %}
+    </select><br />
+    <input type="file" name="model_import" /><br />
+    <select name="date_format">
+        <option selected="selected" value="%Y-%m-%d">YYYY-MM-DD</option>
+        <option value="%Y/%m/%d">YYYY/MM/DD</option>
+        <option value="%m/%d/%Y">MM/DD/YYYY</option>
+        <option value="%d/%m/%Y">DD/MM/YYYY</option>
+    </select>
+    <input type="submit" value="Upload" />
+</form>
+<br/>
+{% if attendees %}
+        <div class="control-group"><h4>Imported Data</h4></div>
+    <table class="table table-striped datatable">
+    <thead><tr>
+        <th>ID</th>
```

Comment:
One thing you may want to include is that Sideboard models have a ``_repr_attr_names`` field, which we sometimes set.  You can see this in a few places in ``models.py`` where we say which columns we actually want to display in a repr at the command line.  Since those are columns which someone has indicated are a good way to indicate what a thing is, they're probably a good candidate for displaying here.

Definitely not required, but if you were looking for a way to determine which fields are valuable on any given model, that's the way to go about it.  (And we could be more diligent about setting that field on our models in the future.)